defaults/main.yml

```diff
@@ -5,6 +5,8 @@ ya_provider_wasi_version: v0.2.2
 ya_provider_wasi_url: "https://github.com/golemfactory/ya-runtime-wasi/releases/download/{{ ya_provider_wasi_version }}/ya-runtime-wasi-linux-{{ ya_provider_wasi_version }}.tar.gz"
 ya_provider_vm_version: v0.2.11
 ya_provider_vm_url: "https://github.com/golemfactory/ya-runtime-vm/releases/download/{{ ya_provider_vm_version }}/ya-runtime-vm-linux-{{ ya_provider_vm_version }}.tar.gz"
+ya_provider_installer_resources_url: "https://github.com/golemfactory/ya-installer-resources/releases/latest/download/resources.tar.gz"
```

Comment:
```suggestion
ya_provider_installer_resources_version: 0.1.5
ya_provider_installer_resources_url: "https://github.com/golemfactory/ya-installer-resources/releases/v{{ ya_provider_installer_resources_version }}/download/resources.tar.gz"
```

Reply:
We can do it in this playbook, but in practice in the playbook updating the network we will always download the `latest` release because these are certificates and whitelist.

Reply:
Then the ya_provider_installer_resources_version should be bumped to the latest version. Sooner or later there will be a situation where certain version of yagna won't work with a certain version of resources. This way we can have idempotent and reproducible environment.